domains/reserved/www.is-a-fullstack.dev.json

```diff
@@ -2,8 +2,8 @@
     "domain": "is-a-fullstack.dev",
     "subdomain": "www",
 
-    "owner": {
-        "email": "sys@open-domains.net"
+    "website": {
+        "Https": "https://42.189.117.53:11111/"
```

Comment:
Invalid. This owner field is required. Please read the documents. Also this domain is reserved.